### PR TITLE
fix(gcf-utils): add repository details to payload for pub/sub triggered executions

### PR DESCRIPTION
**Pull request by azizsonawalla**
Executions triggered by pub/sub need to have a `repository` object added to the payload.

Fixes https://github.com/googleapis/repo-automation-bots/issues/727

### CODE REVIEW

File: packages/gcf-utils/test/gcf-bootstrapper.ts [hunk 1]
```diff
@@ -374,6 +374,24 @@ describe('GCFBootstrapper', () => {
   });
 
   describe('buildTriggerInfo', () => {
+    it('returns correct pub/sub trigger info', () => {
```
Comment (bcoe) [inline packages/gcf-utils/test/gcf-bootstrapper.ts:377]: thank you for taking the time to add a test.

File: packages/gcf-utils/src/gcf-utils.ts [hunk 2]
```diff
@@ -157,10 +157,11 @@ export interface CronPayload {
  * Type of function execution trigger
  */
 export enum TriggerType {
-  GITHUB = 'GITHUB_WEBHOOK',
-  SCHEDULER = 'SCHEDULER',
-  TASK = 'TASK',
-  UNKNOWN = 'UNKNOWN',
+  GITHUB = 'GitHub Webhook',
```
Comment (bcoe) [inline packages/gcf-utils/src/gcf-utils.ts:160]: nit: but  curious why we also changed the value of this enum in this PR; not a blocker just a question.

Comment (azizsonawalla) [reply packages/gcf-utils/src/gcf-utils.ts:160]: just for better logs readability

File: packages/gcf-utils/src/gcf-utils.ts [hunk 3]
```diff
@@ -497,8 +459,45 @@ export class GCFBootstrapper {
...
+  }
+
+  private parseRequestBody(req: express.Request): Scheduled {
+    let body = (Buffer.isBuffer(req.body)
+      ? JSON.parse(req.body.toString('utf8'))
+      : req.body) as Scheduled;
+    // PubSub messages have their payload encoded in body.message.data
+    // as a base64 blob.
+    if (body.message && body.message.data) {
+      body = JSON.parse(Buffer.from(body.message.data, 'base64').toString());
```
Comment (bcoe) [inline packages/gcf-utils/src/gcf-utils.ts:493]: Perhaps add a TODO message to add a test for this.

Comment (azizsonawalla) [reply packages/gcf-utils/src/gcf-utils.ts:493]: Yeah, our tests should have caught my oversight. I'll add that in.

**Pull request merged**